### PR TITLE
docs(plasma-ui-docs): add block "Переносы на SberBox, SberPortal, etc."

### PR DESCRIPTION
**Pull request by Yakutoc**
Мотивация: 

На устройствах для **русского** языка не работает **автоматическая** расстановка переносов. 

Обойти ограничение можно - нужно расставить мягкие переносы - `&shy;`. 

Пример на скриншоте это показывает. 

Снимок экрана 2023-04-07 в 18 07 04
   

:baby_chick: Download canary assets:

[colors--canary.473.4637724191.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.473.4637724191.xml)  
[color_sberHealth_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.473.4637724191.swift)  
[color_sberHealth_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.473.4637724191.kt)  
[color_sberHealth_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.473.4637724191.ts)  
[color_sbermarket_business_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.473.4637724191.swift)  
[color_sbermarket_business_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.473.4637724191.kt)  
[color_sbermarket_business_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.473.4637724191.ts)  
[color_sbermarket_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.473.4637724191.swift)  
[color_sbermarket_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.473.4637724191.kt)  
[color_sbermarket_metro_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.473.4637724191.swift)  
[color_sbermarket_metro_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.473.4637724191.kt)  
[color_sbermarket_metro_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.473.4637724191.ts)  
[color_sbermarket_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.473.4637724191.ts)  
[color_sbermarket_selgros_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.473.4637724191.swift)  
[color_sbermarket_selgros_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.473.4637724191.kt)  
[color_sbermarket_selgros_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.473.4637724191.ts)  
[color_sbermarket_vinlab_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_ios-swift--canary.473.4637724191.swift)  
[color_sbermarket_vinlab_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_kotlin--canary.473.4637724191.kt)  
[color_sbermarket_vinlab_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_vinlab_react-native--canary.473.4637724191.ts)  
[color_sberprime_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.473.4637724191.swift)  
[color_sberprime_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.473.4637724191.kt)  
[color_sberprime_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.473.4637724191.ts)  
[PlasmaTokensColor--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.473.4637724191.swift)  
[shadow_sbermarket_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.473.4637724191.ts)  
[typo_mage_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.473.4637724191.swift)  
[typo_mage_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.473.4637724191.kt)  
[typo_mage_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.473.4637724191.ts)  
[typo_plasma_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.473.4637724191.swift)  
[typo_plasma_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.473.4637724191.kt)  
[typo_plasma_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.473.4637724191.ts)  
[typo_ruler_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.473.4637724191.swift)  
[typo_ruler_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.473.4637724191.kt)  
[typo_ruler_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.473.4637724191.ts)  
[typo_sage_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.473.4637724191.swift)  
[typo_sage_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.473.4637724191.kt)  
[typo_sage_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.473.4637724191.ts)  
[typo_sbermarket_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.473.4637724191.swift)  
[typo_sbermarket_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.473.4637724191.kt)  
[typo_sbermarket_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.473.4637724191.ts)  
[typo_soulmate_ios-swift--canary.473.4637724191.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.473.4637724191.swift)  
[typo_soulmate_kotlin--canary.473.4637724191.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.473.4637724191.kt)  
[typo_soulmate_react-native--canary.473.4637724191.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.473.4637724191.ts)

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-473/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-473/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-473/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-473/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-473/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-473/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-473/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-473/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-473/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-473/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-473/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-473/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-473/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-473/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by neretin-trike — APPROVED** [review]

File: website/plasma-ui-docs/docs/design/typography.mdx [hunk 1]
```diff
@@ -67,6 +69,13 @@ export default function App() {
 }
 ```
 
+### Переносы на SberBox, SberPortal, etc.
+
+На устройствах не работает, **пока что**, автоматическая система переносов для русского языка.
```
Comment (Yeti-or) [inline website/plasma-ui-docs/docs/design/typography.mdx:74]: я пытаюсь выяснить пока когда , давай чуть подождем;
В остальном 🆗

**Review by Yeti-or — APPROVED** [review]

**Pull request merged**